Assets/Scripts/Interactable.cs

```diff
@@ -58,4 +76,29 @@ void setIrritationScore(IrritationReaction irritation)
       Debug.Log($"{npc.name} didn't see you messing with {this}...");
     }
   }
+
+  protected virtual void MouthInteraction()
+  {
+    Debug.Log("MouthInteraction");
+  }
+
+  protected virtual void PawInteraction()
+  {
+    Debug.Log("PawInteraction");
+  }
+
+  protected virtual void BodyInteraction()
+  {
+    Debug.Log("BodyInteraction");
+  }
+
+  public void ShowTooltip()
+  {
+    interactableUI.ShowInteractionTip(true);
+  }
+
+  public void HideTooltip()
+  {
+    interactableUI.ShowInteractionTip(true);
+  }
```

Comment:
These are the convenience methods for the `PlayerController` to use.
If we ever change how the UI works, the `PlayerController` hopefully won't need to change since it's encapsulated here.